src/__mocks__/graceful-fs.js

```diff
@@ -112,6 +112,47 @@ fs.stat.mockImpl(function(filepath, callback) {
   }
 });
 
+fs.statSync.mockImpl(function(filepath) {
+  var node;
+  try {
+    node = getToNode(filepath);
+  } catch (e) {
+    throw e;
+  }
```

Comment:
Yah, you're right, in this context it's not necessary because this is a sync method. I had copied this from the `fs.stat` mock, which WOULD need a try/catch because it needs to errBack.